projects/ADuCM3029_demo_cn0410/RTE/Device/ADuCM3029/ADuCM3029.ld

```diff
@@ -0,0 +1,217 @@
+/* 
```

Comment:
this file looks generated;
should it be here ?

Reply:
The file is a script to configure the memory regions it applies to the controller used and is part of the RTE CMSIS platform folder. I would not remove this and keep the project independent from other packages.